content/rc/how-to/creating-aws-user-redis-enterprise-vpc.md

```diff
@@ -9,14 +9,42 @@ aliases: /rv/how-to/creating-aws-user-redis-enterprise-vpc/
 In most Redis Cloud deployments, the infrastructure of your Redis Cloud subscriptions on AWS is created in dedicated AWS accounts that we manage for you.
 In Redis Cloud Ultimate, you can choose to have this infrastructure on your own AWS accounts.
 You'll want these accounts to be separate from any AWS application accounts,
-and you'll need to create some dedicated IAM entities to let us manage the infrastructure for you.
+and you'll need to create some dedicated IAM entities and then configure your Redis Cloud Account with the necessary details.
+
+The configuration can either be done 'by hand', following the instructions in [Create and Edit a Cloud Account for Redis Cloud Ultimate 
+](rc/how-to/view-edit-cloud-account.md), or via the Cloud API, in which case you'll use the [create cloud account using POST](https://api.redislabs.com/v1/swagger-ui.html#/Cloud%20Accounts/createCloudAccountUsingPOST) call.
 
 In the new AWS account, you need to create:
 
 - An **instance role**
-- A user with an **access key**
+- A user with an **access key** and a password
 - A role that grants **AWS console access**
 
+You have two options for how to create these resources. Choose **one** of them only:
+
+# A. Use Automation (Cloudformation)
```

Comment:
@TobyHFerguson Cool idea, but how can we prevent the CloudFormation template from being out-of-sync with the IAM policies in the documentation?

Reply:
Right now what I do is simply copy/paste the IAM policies directly into the Cloudformation script so, for now, I'd suggest that that would be an additional step required.

It might be possible to use the Fn::Include macro from CloudFormation, but that only pulls in stuff from S3, so now we'd have to add these policies to some S3 bucket somewhere (if only Fn::Include could pull from a github public gist!!!).

Reply:
I've investigated the Fn::Include option more thoroughly and it works reasonably well, so we might wish to use this option. 

This would require that we:

1. Upload two json policy files to S3
2. Modify the cloudformation template to reference those S3 objects

The question remaining is how do we ensure that the objects in S3 are updated on a timely basis - the best way to do this (I believe) is to create an AWS Pipeline that can be triggered by an update in github to those policy files. The pipeline would then copy those files up to S3 (they need to be wrapped to make the json that cloudformation is expecting, but that is a minor operation easily encoded in `jq` thus:

```
cat POLICY | jq '{ PolicyDocument: . }' 
```

where POLICY is the relevant policy that needs to be 'wrapped' ready for cloudformation.

The issue I have is who owns all of this? How can we get this implemented?

Reply:
@erezatiya - any progress here? Does my solution work for you? I'd like to see this getting into the end user docs sooner than later :-)

Reply:
@TobyHFerguson @erezatiya are we waiting on this Cloudformation template to make these instructions valid?

Reply:
@kaitlynmichaelredislabs I'm still in discussion with @cwsteve2117 and @banker on exactly how this will all work together. The CloudFormation template is ready, but we need to determine how we're to manage the public S3 bucket into which it and the (slightly modified) policy files will be placed. We also need to figure out exactly how the (very rare) changes to the policy files will be handled. I think the first step of that will be to pull the policy files out as separate files (right now they're just directly in the documentation!), after which we'll automate the push process (since those same policy files are used by the terraform version of this process, which is a separate pull request I've authored. 

Hope that helps explain where we are right now!

Reply:
@TobyHFerguson could we open an internal JIRA for this issue instead of housing it in a PR on our public repo?

Reply:
@TobyHFerguson Your suggestion sounds feasible, I suggest we discuss this over an internal ticket as mentioned by kaitlyn.